src/Stripe.net/Services/Sources/SourceVerifyOptions.cs

```diff
@@ -0,0 +1,14 @@
+namespace Stripe
+{
+    using System.Collections.Generic;
+    using Newtonsoft.Json;
+
+    public class SourceVerifyOptions : BaseOptions
+    {
+        /// <summary>
+        /// The values needed to verify the source.
+        /// </summary>
+        [JsonProperty("values")]
+        public List<string> Values { get; set; }
```

Comment:
I'm assuming this is supposed to be a string and not an int right?

Reply:
Yep, it's specified as an array of strings, presumably because the values can be more or less anything depending on the source's type.